managed/src/main/java/com/yugabyte/yw/models/CustomerConfig.java

```diff
@@ -104,6 +110,25 @@ public boolean getInUse() {
     return false;
   }
 
+  public ArrayNode getUniverseDetails() {
+    ArrayNode details = Json.newArray();
+    if (this.type==ConfigType.STORAGE){
+      ObjectNode universePayload = Json.newObject();
+      // TODO this would go to the util.java after KMS PR get merged. 
+      for (Universe universe : Backup.getUniverses(this.configUUID)) {
+        UniverseDefinitionTaskParams universeDetails = universe.getUniverseDetails();
+        universePayload.put("name", universe.name);
+        universePayload.put("updateInProgress", universeDetails.updateInProgress);
+        universePayload.put("updateSucceeded", universeDetails.updateSucceeded);
+        universePayload.put("uuid", universe.universeUUID.toString());
+        universePayload.put("creationDate", universe.creationDate.getTime());
+        universePayload.put("universePaused", universeDetails.universePaused);
+        details.add(universePayload);
+      }
+    }
+  return details;
+  }
```

Comment:
The function looks quite strange.
1) I don't have understanding why we need it. Who invokes it?
2) In such implementation it doesn't look good to be moved into the mentioned utility class. It would be great if the utility function doesn't call functions like Customer.get(...), Universe.get(...), Backup.getUniverses(...) etc. IMO, such data should be passed into utility functions as parameters. (yes, the utility class already has some dirty code but don't add more, please) Also don't forget to give the function a good clear name/caption.
3) Format this code

Reply:
@SergeyPotachev the above function is reusable in multiple places ( certificates, KMS configs or wherever we need the universe details ( minimal ) in UI for any configs ) so thought of moving to utility and moved to utility in other PR as Arnav mentioned.

Reply:
OK. @nishantSharma459 Anyway please apply text formatter. Also check that TODO is situated a correct place.

Reply:
The PR will get merge and I will make sure to update this PR before pushing.

Reply:
Can't accept this PR until you merge the mentioned diff. We have quite a lot issues because of blind/uncontrolled merges.